discovery-provider/src/tasks/generate_trending.py

```diff
@@ -78,6 +78,20 @@ def generate_trending(db, time, genre, limit, offset):
         )
         not_deleted_track_ids = set([record[0] for record in not_deleted_track_ids]) # pylint: disable=R1718
 
+        track_created_at = (
```

Comment:
can you just combine this into the above query? since that's already querying tracks. that should give you something like a tuple of ids and created ats

Reply:
wanted to separate out code intention but sure 👍

Reply:
while it's a good idea to separate intention, doing two query here (unless the query optimizer is really really good) is going to be more expensive

Reply:
got it!